.github/workflows/ci.yml

```diff
@@ -131,37 +197,38 @@ jobs:
       # execute migration rules before running tests and self-fixing,
       # so that we know that our codebase is future-ready
       - name: Run PHP CS Fixer with migration rules
-        if: matrix.migration-rules == 'yes'
+        if: matrix.run-migration-rules == 'yes'
         run: php php-cs-fixer fix --config .php-cs-fixer.php-highest.php -q
 
       - name: Disable time limit for tests when collecting coverage
-        if: matrix.calculate-code-coverage == 'yes'
+        if: matrix.code-coverage == 'yes'
         run: sed 's/enforceTimeLimit="true"/enforceTimeLimit="false"/g' phpunit.xml.dist > phpunit.xml
 
       - name: Run tests
-        if: matrix.calculate-code-coverage != 'yes'
+        if: matrix.run-tests == 'yes'
```

Comment:
AND `matrix.code-coverage != 'yes'` , to not execute tests twice?

Reply:
I don't feel the need for that, but I'll add it.